jest.config.js

```diff
@@ -9,7 +10,7 @@ module.exports = {
     '^.+\\.tsx?$': 'ts-jest',
   },
   testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$',
-  moduleDirectories: ['node_modules', 'src'],
```

Comment:
How this is necessary? https://jestjs.io/docs/en/configuration#moduledirectories-arraystring

Reply:
Same reason it's changed in webpack config

From the doc you linked:
> An array of directory names to be searched recursively up from the requiring module's location

guess what happens when there's src in `node_modules/some_dep/src`

Reply:
Really? Why does it have precedence a node_modules than our own `src`

Reply:
don't know, who wrote it, must have had a reason